docs/troubleshootings/troubleshooting-agic-addon-identity-not-found.md

```diff
@@ -0,0 +1,23 @@
+## Troubleshooting: AGIC addon, identity not found
+
+### Illustration
+AGIC fails to start due to "Identity not found" even though the required permissions are provided.
```

Comment:
How about changing this to ?
`AGIC fails to start due to "Identity not found" even though the User Managed Identity to the AKS nodes is correctly assigned and it has the right permissions.`